pkg/controller/controlplanecerts/controlplanecerts_controller.go

```diff
@@ -263,8 +263,9 @@ func (r *ReconcileControlPlaneCerts) generateControlPlaneCertsSyncSet(cd *hivev1
 	cdLog.Debug("generating syncset for control plane secrets")
 	syncSet := &hivev1.SyncSet{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      GenerateControlPlaneCertsSyncSetName(cd.Name),
-			Namespace: cd.Namespace,
+			Name:        GenerateControlPlaneCertsSyncSetName(cd.Name),
+			Namespace:   cd.Namespace,
+			Annotations: map[string]string{constants.SyncSetMetricsGroupAnnotation: "cp-certs"},
```

Comment:
changes in controllers can go in as a separate commit to document the label already present on the generated syncsets